charts/uplink/templates/pdb.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if (gt (int .Values.replicaCount) 1) }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "uplink.fullname" . }}
+  labels:
+    {{- include "uplink.labels" . | nindent 4 }}
+spec:
+  {{- with .Values.poddisruptionbudget.minAvailable }}
```

Comment:
`.Values.poddisruptionbudget` is undefined causing the error `Error: Failed to render chart: exit status 1: Error: template: uplink/templates/pdb.yaml:9:18: executing "uplink/templates/pdb.yaml" at <.Values.poddisruptionbudget.minAvailable>: nil pointer evaluating interface {}.minAvailable`.

You might want to look at  https://github.com/jenkins-infra/helm-charts/pull/822/files and add the corresponding tests (first) and fix